### DISCUSSION TITLE
Sample codes do not run in my environment

### DISCUSSION THREAD

**Discussion by yusuke-shimono** [NTNU-IHB/PythonFMU#173] [Q&A]
I'm interested in PythonFMU, and try to use it in my PC.
I installed PythonFMU and build FMUs by sample codes (examples/demoslave.py and Example in README.md）

However, they don't run in 1D-simulation tools like as MATLAB.

Could you tell me how to check FMU is built correctly, and the environment required to run FMU which build by PythonFMU?

Best regards,

**Comment by markaren** [comment]
Python and any scripts it needs to be available globally. Furthermore, the Python version potentially used in the host application needs to match the one linked with PythonFMU. This makes it difficult or even impossible to run in some hosts that themselves links with Python.

Linux is tricky as the Python environment must match exactly. If installed with pip that would be x64 v3.9. On Windows, x64 3.x >= 3.6 should work.

You could test other importers to see if MATLAB is the issue (i.e. if it bundles its own incompatible Python).

**Reply by yusuke-shimono** [reply]
Thank you very mutch for your replying and I sincerely apologize for the late reply.

I tried to run my FMU on global python environment, but it couldn't run.

I tried to run it on Windows. (Windows 10)
I used the following python and PythonFMU version.
Python 3.9.13
pythonfmu  0.6.2

I installed pyton in C:\Program Files\Python39
and I added this folder to the top of the PATH system environment variable.

I built a sample script (https://github.com/NTNU-IHB/PythonFMU#example) the following command.
pythonfmu build -f pythonslave.py

The build was successful and the attached FMU file was generated. 
Please find an attached file. (Please unzip this file)

[PythonSlave.zip](https://github.com/NTNU-IHB/PythonFMU/files/11059430/PythonSlave.zip)

However, I coundn't run it on any FMI tools.

Could you tell me what kind of environment you are running FMU in?
Can you run the attached FMU file normally in that environment?

I would be very grateful if you could let me know what the cause is.

Best regards.
